pom.xml

```diff
@@ -458,7 +458,7 @@
             <excludes>
               <exclude>.gitignore</exclude>
               <exclude>.repository/</exclude>
-              <exclude>.idea/</exclude>
+              <exclude>**/.idea/**</exclude>
```

Comment:
this change should not be necessary. the `.idea/` folder should only get created at the project root level. any nested files and folders are already covered. maybe a mishap when setting up IntelliJ?

Reply:
Yea, I think it's extra files in my IntelliJ setup.  I'll rebased my changes.